fairscale/experimental/wgit/__init__.py

```diff
@@ -3,7 +3,7 @@
 # This source code is licensed under the BSD license found in the
 # LICENSE file in the root directory of this source tree.
 
-from .cli import main
-from .weigit_api import WeiGit
+from .repo import Repo
+from .version import __version_tuple__
 
-__version__ = "0.0.1"
+__version__ = ".".join([str(x) for x in __version_tuple__])
```

Comment:
after this line, perhaps add

__all__ = []

so that `import *` won't import anything by default to avoid polluting the name space. That's a convention we use in fairscale.